float64[] position_new_1
bool isReady_new_1
float64[] position_new_2
bool isReady_new_2